### PR TITLE
Added polish translations

### PR DESCRIPTION
**Pull request by Przemro**

### CODE REVIEW

File: stringtable.xml [hunk 1]
```diff
@@ -14,9 +14,11 @@
       </Key>
       <Key ID="STR_f_param_loadouts">
         <English> Loadouts </English>
+        <Polish> Wyposażenia </Polish>
       </Key>
       <Key ID="STR_f_param_timeOfDay">
         <English> Time of Day </English>
+        <Polsih> Pora dnia </Polsih>
```
Comment (cptnnick) [inline stringtable.xml]: Spelling mistake

File: stringtable.xml [hunk 2]
```diff
@@ -215,27 +226,34 @@
...
         <English> Click on the map to set HALO coordinates. </English>
+        <Polish> Kliknij na mape by ustawić miejsce skoku HALO</Polish>
       </Key>
       <Key ID="STR_f_mapClickHALODone">
         <English> You have been moved to the selected location. </English>
+        <Polish> Zostales przeniesiony do wybranej lokacji. </Polish>
       </Key>
       <Key ID="STR_f_JIP_reinforcementOptionsAction">
         <English> Select Reinforcement Options</English>
+        <Polsih> Wybierz opcje wsparcia</Polsih>
```
Comment (cptnnick) [inline stringtable.xml]: spelling mistake

**Comment by cptnnick** [comment]
Few small spelling mistakes, and most words between parenthesis weren't capitalized while the English  strings were.

**Comment by Wolfenswan** [comment]
@Przemro  needs to be pulled into dev-branch.

**Comment by ferstaberinde** [comment]
Hi @Przemro - please can you re-submit this PR to the Dev branch? Thanks :)

**Pull request closed without merging**